fix: [MTM-58369] add missing release notes for UI (#1870)
SoftwareAG/c8y-docs: content/change-logs/platform-services/ui-c8y-10-18-500-11-issue-with-c8y-cli-task-for-compiling-po-strings-to-json-on-windows.md

---
date: 

> title: issue wuth c8y cli task for compiling po strings to json on windows

Reviewer: ```suggestion
title: Issue with c8y CLI task for compiling po strings to json on Windows
```

> title: issue wuth c8y cli task for compiling po strings to json on windows

Reviewer 2: ```suggestion
title: Localization strings are correctly compiled to JSON on windows
```

> title: Localization strings are correctly compiled to JSON on windows using c8ycli command

Reviewer 3: ```suggestion
title: Localization strings are correctly compiled to JSON on Windows using c8ycli command
```
